common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -125,6 +135,15 @@ public void onInventoryKeyPress(InventoryKeyPressEvent event) {
 
     private void tryAutoSearch(AbstractContainerScreen<?> abstractContainerScreen) {
         if (!autoSearching) return;
+        if (currentSearchableContainerType == SearchableContainerType.GUILD_BANK) {
+            if (System.currentTimeMillis() - guildBankLastSearch < GUILD_BANK_SEARCH_DELAY) {
+                Managers.TickScheduler.scheduleLater(
+                        () -> tryAutoSearch(abstractContainerScreen),
+                        (int) (GUILD_BANK_SEARCH_DELAY - (System.currentTimeMillis() - guildBankLastSearch)) / 20);
```

Comment:
`System.currentTimeMillis() - guildBankLastSearch` needs to be extracted to prevent weird races where you could get - ticks.